symfony 5 support

Bast1onCZ opened the pull request:
I changed the necessary things to be able to run the library in symfony 5. Please publish this package.

Bast1onCZ committed:
symfony 5 support

Bast1onCZ committed:
fix GraphQLConfigureCommand.php - added status codes in command execute

Bast1onCZ committed:
implemented replacement

Bast1onCZ committed:
fix replace

Bast1onCZ committed:
renamed back to youshido

arendjantetteroo commented:
Works for me. 

Complains about 2 more items regarding the controllers.
The "Youshido\GraphQLBundle\Controller\GraphQLController" class extends "Symfony\Bundle\FrameworkBundle\Controller\Controller" that is deprecated since Symfony 4.2, use "Symfony\Bundle\FrameworkBundle\Controller\AbstractController" instead.

and

The "Youshido\GraphQLBundle\Controller\GraphQLExplorerController" class extends "Symfony\Bundle\FrameworkBundle\Controller\Controller" that is deprecated since Symfony 4.2, use "Symfony\Bundle\FrameworkBundle\Controller\AbstractController" instead.

Bast1onCZ commented:
You are right. Please fork my repo and create a pull request

Bast1onCZ committed:
changed project name

Bast1onCZ committed:
init

Bast1onCZ committed:
fix AGraphQLController.php

Bast1onCZ committed:
fixed AGraphQLController.php and added service container autowire alias to services.xml

Bast1onCZ committed:
removed dump from AGraphQLController.php

Bast1onCZ committed:
fixed license

Bast1onCZ committed:
fixed namespaces

Bast1onCZ committed:
project wide code review

Bast1onCZ committed:
fixed AGraphQLExplorerController.php

Bast1onCZ committed:
fixed Processor.php not to throw irrelevant errors in isServiceReference

Bast1onCZ committed:
implemented isGranted and denyAccessUnlessGranted for GraphQLRequest.php

Bast1onCZ committed:
fix denyAccessUnlessGranted doc

Bast1onCZ committed:
fixed explorer.html.twig to not send Authorization token with empty 'Bearer'

Bast1onCZ committed:
simplified AGraphQLAuthenticator.php and added bearerToken array key to credentials result

Bast1onCZ committed:
fixed explorer.html.twig

Bast1onCZ committed:
fixed AGraphQLAuthenticator.php bearerMatch

Bast1onCZ committed:
added uuid regex checking in UuidType.php

Bast1onCZ committed:
fixed AGraphQLAuthenticator.php bearer regex

Bast1onCZ committed:
implemented ATranslatableEntityType.php to automatically translate fields to current language

Bast1onCZ committed:
renamed TOrderableType to TIndexableType

Bast1onCZ committed:
updated utils

Bast1onCZ committed:
processEntityTranslatedField fixed:
- persist translatable if not managed yet (because of doctrine event feeding deps)

Bast1onCZ committed:
fixed ATranslationType.php to return language code in locale field

Bast1onCZ committed:
commented GraphQLBundle.php class

Bast1onCZ committed:
added localisation graphql object types

Bast1onCZ committed:
removed localisation types

Bast1onCZ committed:
fixed NonEmptyListType.php

Bast1onCZ committed:
implemented non null locale in ATranslationType.php

Bast1onCZ committed:
changed GraphQLRequest::processEntityIdConnectionField to find entity right from entity manager without using repository

Bast1onCZ committed:
deprecated LoggedUserVoter.php

Bast1onCZ committed:
implemented QueryOperationVoter.php and MutationOperationVoter.php, changed security constants in SecurityManagerInterface.php

Bast1onCZ committed:
implemented secured fields & SecuredFieldVoter.php

Bast1onCZ committed:
changed AFreeOperationVoter to FreeFieldVoter

Bast1onCZ committed:
removed unused voters

Bast1onCZ committed:
fixes

logosur commented:
Hello,

Nobody taking this pull request? 

Regards.

Bast1onCZ committed:
updated LICENSE.md

Bast1onCZ committed:
changed ASimpleEnumType.php to uppercase keys
